### PR TITLE
XSOAR (Independent Publisher)

### PR DESCRIPTION
**Pull request by landonc**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to COVID delays, unanticipated team leaves, and high volume of connector submissions. 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

connection_test_success
flow_success

### CODE REVIEW

File: independent-publisher-connectors/XSOAR/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+settings.json
```
Comment (sriyen-msft) [inline independent-publisher-connectors/XSOAR/.gitignore:1]: @landonc Please remove this file

File: independent-publisher-connectors/XSOAR/Readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+You will need the following to proceed:
+* A XSOAR with an integration instance available over HTTPS via [TODO HERE].
+* An API key for the integration instance
+
+## Supported Operations
+The connector supports the following operation:
+* `Send to XSOAR`: Send JSON data to XSOAR
+
+## Obtaining Credentials
+API keys are created by configuring an integration instance within XSOAR.
```
Comment (sriyen-msft) [inline independent-publisher-connectors/XSOAR/Readme.md:16]: @landonc Can you please provide more details in step-by-step format to obtain an API key from XSOAR

File: independent-publisher-connectors/XSOAR/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  "host": "demisto.live",
+  "basePath": "/instance/execute",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.paloaltonetworks.com/"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/XSOAR/apiDefinition.swagger.json:25]: @landonc Please provide specific website URL of the connector service like below:
https://start.paloaltonetworks.com/sign-up-for-community-edition.html

File: independent-publisher-connectors/XSOAR/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "401": {
+            "description": "API key is missing or invalid",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/XSOAR/apiDefinition.swagger.json:64]: @landonc No response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

File: independent-publisher-connectors/XSOAR/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        "title": "SetHostURL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('domain_name')/instance/execute/@connectionParameters('instance_name')",
+          "x-ms-apimTemplate-operationName": [
+            "SendToXSOAR"
+          ]
+        }
+      }
+    ],
+    "publisher": "Landon Chelf"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/XSOAR/apiProperties.json:57]: @landonc Please add stackOwner property as well.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by samrat-gavale — APPROVED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

File: independent-publisher-connectors/XSOAR/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,100 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Provides the ability to submit content to XSOAR integrations when using instance.execute.external to expose integration instances over HTTPS. Tested to work with the Generic Webhook integration.",
+    "version": "1.0.0",
+    "title": "XSOAR",
+    "contact": {
+      "name": "Landon Chelf",
+      "email": "landon.chelf@fisglobal.com"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/XSOAR/apiDefinition.swagger.json:9]: @landonc Syntax error: Can you please add a comma after email value ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/XSOAR/apiDefinition.swagger.json (hunk 6 above)
Comment (sriyen-msft) [reply independent-publisher-connectors/XSOAR/apiDefinition.swagger.json:9]: @landonc Please raise a PR to fix it, thanks in advance.

**Comment by sriyen-msft** [comment]
@landonc Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**